components/firebase_admin_sdk/actions/update-document/update-document.mjs

```diff
@@ -2,17 +2,20 @@ import common from "../common/base.mjs";
 
 export default {
   ...common,
-  key: "firebase_admin_sdk-update-documents",
+  key: "firebase_admin_sdk-update-document",
```

Comment:
Are users able to update their workflows despite the component key being changed?